VideoApi/VideoApi/Controllers/ConferenceController.cs

```diff
@@ -536,6 +536,23 @@ private async Task EnsureAudioFileExists(Conference conference)
             return true;
         }
 
+        public async Task<AudioPlatformServiceResponse> CreateAudioApplicationAsync(Guid hearingRefId)
+        {
+            try
+            {
+                return await _audioPlatformService.CreateAudioApplicationAsync(hearingRefId);
+            }
+            catch (AudioPlatformException ex)
+            {
```

Comment:
Better to check if we have application and then Create or carry on.. instead of relying on exceptions

Reply:
Maybe but this is an edge case and the check would add an additional call to every booking?